docs/en-US/golos.vesting_contract.md

```diff
@@ -291,17 +289,20 @@ Transaction must be signed by the `owner` account.
 
 To perform the action, it is required that the balance of all vesting, including the delegated one, equals to zero.
 
-## procwaiting
-The `procwaiting` action is used to speed up withdrawing of vestings and returning of delegations. The signature of this action is:
+## paydelegator
```

Comment:
Action `vesting::paydelegator` missing now. Information about `vesting::procwaiting` is worth leaving.